app/spike/internal/cmd/operator/recover.go

```diff
@@ -146,8 +146,8 @@ func newOperatorRecoverCommand(
 				err := os.WriteFile(filePath, []byte(out), 0600)
 
 				// Security: Hint gc to reclaim memory.
-				encodedShard = ""
-				out = ""
+				_ = encodedShard
+				_ = out
```

Comment:
This is intentional; we should not revert this. -- we can add an lint ignore comment instead (which is already there in one of the PRs).

Reply:
If this one overrides existing code, I'll re-override it :)